politeiawww/www.go

```diff
@@ -53,9 +53,13 @@ const (
 )
 
 var (
-	// ErrSessionUUIDNotFound is emitted when a UUID value is not found
-	// in a session and indicates that the user is not logged in.
-	ErrSessionUUIDNotFound = errors.New("session UUID not found")
+	// ErrSessionNotFound is emitted when a session is not found and indicates
+	// that the user is not logged in.
+	ErrSessionNotFound = errors.New("session not found")
```

Comment:
These are in main so they don't need to be exported.  Not sure why `ErrSessionUUIDNotFound` was, but it shouldn't have been.

Reply:
Fixed.